e2-studio-star-rx72n-firmware/libs/rx_hal/src/rx_mtu.c

```
@@ -349,7 +341,12 @@ static rx_err_t internal_calculate_period(const uint32_t frequency_hz, uint16_t*
     return k_rx_err_invalid_arg;
   }
 
-  period_calc = pclka / (k_mtu_period_divisor * frequency_hz);
+  /* For PWM mode 1 (triangle wave):
+   * Period = PCLKA / (2 * frequency)
+   * PCLKA = 120 MHz
+   */
+  const uint32_t pclka       = k_pclka_hz;
```

Clang (14.0.6)

[warning] variable 'pclka' is not initialized

(cppcoreguidelines-init-variables)